main_test.txtar

```diff
@@ -282,6 +282,9 @@ stderr '^<err: bitwise not of 1.1>$'
 !grol -quiet -c '"abc" * -3'
 !stderr 'panic'
 
+# time parse round trip
+grol -quiet -c 'now=time(); time_parse(time_info(now).str) == now'
```

Comment:
I would have expected test about:
- date computing from string
- date with a timezone
- using local timezone

Reply:
Exactly, manual tests are good tests if we forget to add them to _test, then the work is lost for next refactoring

Reply:
I'll get to #105 but right now working on a pretty huge change (wip) #206